From 99379543cdfc7ed181aa7a5223c28eb747b3302d Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 6 Jun 2017 00:13:23 +0800
Subject: [PATCH 1848/5242] MLK-15008-1 ARM64: dts: freescale: imx8qxp: update
 cpu-freq opp table

commit  4dd2dd0ecbb9ad2ddf6878230d53cbddd299f7c5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update i.MX8QXP cpu-freq opp table according to SCFW setting,
and move the cpu-freq opp table to soc dtsi instead of board
dtb, as it is SoC feature.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi |    5 -----
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi   |    8 ++++++++
 2 files changed, 8 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi
index 4ebc159..2a2f155 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi
@@ -28,11 +28,6 @@
 			reg = <0x0 0x0>;
 			enable-method = "psci";
 			next-level-cache = <&A35_L2>;
-			operating-points = <
-				/* kHz    uV */
-				1000000 1150000
-			>;
-			clocks = <&clk IMX8QXP_A35_DIV>;
 		};
 
 		A35_1: cpu@1 {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 06fe207..29c66fc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1005,3 +1005,11 @@
 		power-domains = <&pd_hifi>;
 	};
 };
+
+&A35_0 {
+	operating-points = <
+		/* kHz    uV */
+		1200000 1150000
+	>;
+	clocks = <&clk IMX8QXP_A35_DIV>;
+};
-- 
1.7.9.5

